File: redsheet-portal/src/app/database-models/v-lite-meeting-management-timing.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vLiteMeetingManagementTiming {
    constructor(vLiteMeetingManagementTiming?: vLiteMeetingManagementTiming )  {
        let properties = [ "liteMeetingManagementId", "liteMeetingManagementTimingId", "negotiationPhase", "negotiationPhaseId", "process", "sortOrder", "tacticId", "tacticName", "time",];
        for (let property of properties)
            if (vLiteMeetingManagementTiming && vLiteMeetingManagementTiming[property])
                this[property] = vLiteMeetingManagementTiming[property];
    }
 
	liteMeetingManagementId : number =   0 ;
 
    @range(1,2147483647)
	liteMeetingManagementTimingId : number =   undefined;
 
    @required()
    @maxLength(100)
	negotiationPhase : string =   undefined;
 
    @range(1,2147483647)
	negotiationPhaseId : number =   undefined;
 
    @required()
    @maxLength(1000)
	process : string =   undefined;
 
    @range(1,2147483647)
	sortOrder : number =   undefined;
 
    @range(1,2147483647)
	tacticId : number =   undefined;
 
    @required()
    @maxLength(200)
	tacticName : string =   undefined;
 
    @required()
	time : Date =   undefined;


}

File: RedSheet.DbEntities/Models/vLiteMeetingManagementTiming.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vLiteMeetingManagementTimings", Schema = "dbo")]
	[Serializable]
	public partial class vLiteMeetingManagementTiming  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int LiteMeetingManagementId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int LiteMeetingManagementTimingId { get; set; } 
 
	[Required] 
	[MaxLength(100)]
	public string NegotiationPhase { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int NegotiationPhaseId { get; set; } 
 
	[Required] 
	[MaxLength(1000)]
	public string Process { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int SortOrder { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int TacticId { get; set; } 
 
	[Required] 
	[MaxLength(200)]
	public string TacticName { get; set; } 
 
	[Required]
	public TimeSpan Time { get; set; } 

	public vLiteMeetingManagementTiming()
	{
    }

    }
}

